source/en/skins-system.rst

```diff
@@ -0,0 +1,163 @@
+Skins system
+------------
+
+On this page you'll find information about available endpoints of Ely.by's service of skins system. You can use any of
+them as an secondary or primary source of skins for your project.
+
+Ely.by's service of skins system provides `proxying of textures from Minecraft premium users <#textures-proxy>`_,
+which means that when using this service players will see both premium Minecraft users' skins and Ely.by users' skins.
+
+We strive to comply with the official skins system and do not support ears and HD-skins. The system supports capes,
+but doesn't allow players to wear them on their own.
+
+If you have suggestions for improvements of existing functionality, please
+`create a new Issue <https://github.com/elyby/chrly/issues/new>`_ in the
+`Chrly project repository <https://github.com/elyby/chrly>`_.
+
+Requests URLs
+=============
+
+.. note:: You can find more detailed information about the implementation of the skins system server in the
+          `Chrly project repository <https://github.com/elyby/chrly>`_.
+
+The skins system is placed on the domain :samp:`http://skinsystem.ely.by`. The :samp:`nickname` parameter
+is case-insensitive.
```

Comment:
"The :samp:`nickname` parameter is case-insensitive."

Feels a bit random/weird to learn about this in this paragraph.